{"name": "test_wait_more_30_sec[chromium]", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: Page.goto: net::ERR_CONNECTION_REFUSED at http://127.0.0.1:8000/\nCall log:\nnavigating to \"http://127.0.0.1:8000/\", waiting until \"load\"", "trace": "get_browser = <Browser type=<BrowserType name=chromium executable_path=/home/oleg/.cache/ms-playwright/chromium-1117/chrome-linux/chrome> version=125.0.6422.26>\nrequest = <SubRequest 'desktop_app' for <Function test_wait_more_30_sec[chromium]>>\n\n    @fixture(scope='session')\n    def desktop_app(get_browser, request):\n        base_url = request.config.getini(\"base_url\")\n        app = App(get_browser, base_url=base_url, **BROWSER_OPTIONS)\n>       app.goto(\"/\")\n\n../conftest.py:67: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../page_object/application.py:31: in goto\n    self.page.goto(self.base_url + endpoint)\n../../.venv/lib/python3.10/site-packages/playwright/sync_api/_generated.py:8686: in goto\n    self._sync(\n../../.venv/lib/python3.10/site-packages/playwright/_impl/_page.py:519: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n../../.venv/lib/python3.10/site-packages/playwright/_impl/_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n../../.venv/lib/python3.10/site-packages/playwright/_impl/_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7df1bf8a13f0>\ncb = <function Channel.send.<locals>.<lambda> at 0x7df1bed783a0>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.Error: Page.goto: net::ERR_CONNECTION_REFUSED at http://127.0.0.1:8000/\nE           Call log:\nE           navigating to \"http://127.0.0.1:8000/\", waiting until \"load\"\n\n../../.venv/lib/python3.10/site-packages/playwright/_impl/_connection.py:514: Error"}, "attachments": [{"name": "log", "source": "010ac14c-3b15-4893-bdb8-84324e4f3805-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chromium'"}], "start": 1721912118985, "stop": 1721912118985, "uuid": "2b72c480-cc17-4314-babf-16d9489c5713", "historyId": "217293120d8e59957a3dacd0c0717110", "testCaseId": "bb957d92fc56f1b2ad69d3db969e031c", "fullName": "lesson_23.tests.test_demo#test_wait_more_30_sec", "labels": [{"name": "parentSuite", "value": "lesson_23.tests"}, {"name": "suite", "value": "test_demo"}, {"name": "host", "value": "acernb001"}, {"name": "thread", "value": "132215-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "lesson_23.tests.test_demo"}]}